config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get ':page' => 'pages#show'
```

Comment:
@marderer let's use one-route-one-action way for information pages. Such dynamic way isn't easy to support, especially when you have different preloaded data on different pages.

What do you think?

Reply:
@marderer why didn't you answer on the Q? am I strange guy? 😄